pkg/podgrouper/podgrouper/plugins/knative/knative.go

```diff
@@ -46,6 +46,10 @@ func NewKnativeGrouper(client client.Client, defaultGrouper *defaultgrouper.Defa
 	}
 }
 
+func (g *KnativeGrouper) Name() string {
```

Comment:
I dont manage to understand why do we need that name.

I dont see its being used at all